pkg/remoteresolution/resolver/bundle/resolver.go

```diff
@@ -69,10 +70,18 @@ func (r *Resolver) GetSelector(context.Context) map[string]string {
 
 // Validate ensures reqolution request spec from a request are as expected.
 func (r *Resolver) Validate(ctx context.Context, req *v1beta1.ResolutionRequestSpec) error {
-	return bundle.ValidateParams(ctx, req.Params)
+	if len(req.Params) > 0 {
+		return bundle.ValidateParams(ctx, req.Params)
+	}
+	// Remove this error once validate url has been implemented.
+	return errors.New("cannot validate request. the Validate method has not been implemented.")
```

Comment:
Could we add test cases for those failure usage?

Reply:
Sure.